ballerina/tests/MedicalItemClient.bal

```diff
@@ -0,0 +1,79 @@
+import ballerinax/mysql;
```

Comment:
Shall we add headers for the test files as well

Reply:
Fixed in [7a6e9d7](https://github.com/ballerina-platform/module-ballerina-persist/pull/1/commits/7a6e9d7ffc80a3a6cd08d65b64ac119899a90bc7)